campari/RomanASP.py

```diff
@@ -252,6 +252,8 @@ def main():
     parser.add_argument("--add-truth-to-lc", action=argparse.BooleanOptionalAction, default=False, help="If True, "
                         "will add the truth fluxes from ou2024 to the lightcurve output. Default False.")
 
+    parser.add_argument("--nprocs", type=int, default=10, help="Number of processes to use. Default 10.")
```

Comment:
I would have gone with the singular for the variable name: `nproc`
Is there a SLURM or Python Pool convention for this being plural `nprocs`?